<ui version="4.0" >
 <class>FileBrowserWidget</class>
 <widget class="QWidget" name="FileBrowserWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>240</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="searchLabel" >
       <property name="text" >
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="searchLineEdit" />
     </item>
     <item row="0" column="2" >
      <widget class="QToolButton" name="clearSearchButton" >
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QToolButton" name="browseButton" >
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeView" name="treeView" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
